classes/02class/exercises/c02-network02/danielvca/README.md

```diff
@@ -0,0 +1,34 @@
+﻿
+# c02-network02
+
+## Commands Execution Output
+
+- Answers for `10.0.0.0/16`:
+
+The network 10.0.0.0/16 has 65534 hosts.
+
```

Comment:
Nice 😄 !
Maybe a mention of the CIDR you've used ?
In this instance from /16 to a /18 !